### ISSUE TITLE
Consistently getting "Specify which project file to use..."

### ISSUE THREAD

**Issue by threadz** [hvanbakel/CsprojToVs2017#265]
After installing and attempting to run,  I'm consistently getting...
"Specify which project file to use because this 'D:\ATLASSVN\Common\source\Utilities\Utilities' contains more than one project file."

My command line is: _dotnet migrate-2019 wizard "Utilities.csproj"_

The directory actually only contains one .csproj file.   I've attached my .csproj file and a screenshot of my command window.

If there anything I'm obviously doing wrong?

[Utilities.csproj.zip](https://github.com/hvanbakel/CsprojToVs2017/files/3628713/Utilities.csproj.zip)

![2019-09-19 11_12_24-Common- Microsoft Visual Studio (Administrator)](https://user-images.githubusercontent.com/5687119/65193459-54f5ef80-dacf-11e9-8d3c-6dffd1bd709b.png)

**Comment by hvanbakel** [comment]
Your directory is called Utilities.csproj? I would need to test but why
does your folder have an extension :)

Not saying we shouldn't fix that but it seems weird

**Comment by threadz** [comment]
No. Project file in the Utilities directory is called Utilities.csproj.  I was referencing the project file directly from the command line.

It's a .NET Framework 4 class library.

Seemed a weird error message to me too.

**Comment by andrew-boyarshin** [comment]
@threadz It's strange, to the point of being weird.
1. There is currently no such string in this project. It's not the message of `dotnet-migrate-2019`.
2. Even if there were, the wizard should list all ambiguous variants (in case there are less than 5).

1. What is your version of `project2015to2017.migrate2019.tool` (`dotnet tool list -g`)?
2. Try turning on extreme log verbosity. `dotnet migrate-2019 -v diag wizard "Utilities.csproj"`
3. Try specifying ".\Utilities.csproj" or full path. Try not specifying anything, it should pick that file, since it's the only project/solution file in the current working directory.

**Comment by threadz** [comment]
Wow, that is odd then.  Here's the screenshot from the CMD window with your suggestions...

![2019-09-19 15_03_03-C__WINDOWS_system32_cmd exe](https://user-images.githubusercontent.com/5687119/65210271-b8dbe080-daee-11e9-848f-aef2f2d4934a.png)

**Comment by hvanbakel** [comment]
Apparently this error is also run when running `dotnet ef`
https://github.com/aspnet/EntityFrameworkCore/issues/11315
